app/controllers/reservations_controller.rb

```diff
@@ -382,7 +382,7 @@ def checkout_email
   end
 
   def checkin_email
```

Comment:
this is done except it made a few tests fail and I'm not quite sure why, so I'm waiting to commit it